woocommerce/google-listings-and-ads
# Replace storybook deps in favor of woocommerce-grow/storybook

### Changes proposed in this Pull Request:

This PR replaces the dependencies regarding storybook in favor of the Storybook tool in the new [WooCommerce Grow Repo ](https://github.com/woocommerce/grow)

### Detailed test instructions:

1.  Checkout this PR
2. Run `npm i`
3. Run `npm run storybook`, `npm run storybook:build` and `npm run storybook:deploy` and see that they are working as expected. 


### Additional details:

The deployed Storybook is here
https://woocommerce.github.io/google-listings-and-ads/?path=/story/tree-select-control--base

The ugly package name will be replaced by the NPM package when we have it available.
